components/lib/tooltip/Tooltip.js

```diff
@@ -150,7 +150,7 @@ export const Tooltip = React.memo(
                     elementRef.current.style.pointerEvents = 'none';
                 }
 
-                if ((isMouseTrack(currentTargetRef.current) || position == 'mouse') && !containerSize.current) {
+                if (((isMouseTrack(currentTargetRef.current) || position == 'mouse') && !containerSize.current) || (isMouseTrack(currentTargetRef.current) || position == 'mouse')) {
```

Comment:
Since we are calling isMouseTrack method twice here break it out to a const on the line above.